### DISCUSSION TITLE
Colors in extension

### DISCUSSION THREAD

**Discussion by jalanh11640** [ryanluker/vscode-coverage-gutters#449] [Q&A] [answered]
I really like this extension.  The one issue I'm having is that for the life of me I can't seem to get the colors to change.  If it matters, I'm developing C++ in a container.

Here are the settings I'm using and I don't see any difference when I set them vs when I don't. I've tried putting them in settings.json and in devcontainer.json.

				"coverage-gutters.highlightdark": "rgba(0, 255, 64, 0.2)",
				"coverage-gutters.noHighlightDark": "rgba(255, 0, 0, 0.2)",
				"coverage-gutters.partialHighlightDark": "rgba(255, 235, 0, 0.2)",
				"coverage-gutters.showLineCoverage": false,
				"coverage-gutters.showRulerCoverage": true

I'm red/green colorblind so the default colors are virtually impossible for me distinguish.

Thanks for any help!
Jeff

**Comment by ryanluker** [comment]
@jalanh11640 Thanks for the discussion item!
We recently revamped how the colours work, and it should be much better in an upcoming release.
https://github.com/ryanluker/vscode-coverage-gutters/pull/436

In the meantime, you can see where we originally updated the defaults to account for colourblindness but this might not apply to everyone?
https://github.com/ryanluker/vscode-coverage-gutters/issues/330

**Reply by jalanh11640** [reply]
Thanks, do you know when the next release will be?

**Reply by ryanluker** [reply, marked as answer]
I should be able to do one today that includes the colour changes!

**Reply by ryanluker** [reply]
@jalanh11640 Release was completed a few weeks ago, you should be able to use the colours now.
